docs/feature-flags/sdks/go.md

```diff
@@ -120,8 +120,9 @@ The `getStringAssignment` function takes two required and one optional input to
 Additional functions are available:
 
 ```
-getBoolAssignment(...)
-getNumericAssignment(...)
+GetBoolAssignment(...)
+GetNumericAssignment(...)
+GetJSONStringAssignment(...)
```

Comment:
I couldn't find any function for getting parsed JSON

https://github.com/Eppo-exp/golang-sdk/blob/8dc51f3df2598b49b9ec47e0dbfe0e49dec82472/eppoclient/client.go#L36-L53

Should we add it in .. or forget about it until someone asks for it?

Also looks like JSON methods are missing in dot net:

https://github.com/Eppo-exp/dot-net-server-sdk/blob/35c9f0b3f464c43650c9b0f7d5ef1d982d7857ec/dot-net-sdk/EppoClient.cs#L32-L46

Reply:
I think your approach of filing low-priority tickets to eventually do this is a good one.

Reply:
Sounds good. Added it to the triage: https://linear.app/eppo/issue/FF-1003/golang-missing-getparsedjsonassignment